src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -108,7 +110,9 @@
                         <p>None of the used translation messages are defined for the given locale.</p>
                     </div>
                 {% else %}
-                    {{ helper.render_table(messages_defined) }}
+                    {% block definedMessagesTable %}
```

Comment:
`Table` is an implementation detail (maybe some day we use two tables or a list, ...) so let's remove it from the block name. And let's use the common case used in Twig: `{% block defined_messages %}`